### PR TITLE
Feat: Pages added

### PR DESCRIPTION
**Pull request by yashpatel1797**
pages added 
1. Homepage
2. Cart management
3. Wishlist management
4. Product listing page

[Live link](https://joyful-rolypoly-31b73d.netlify.app/)

Product Listing page.
image

wishlist page.
image

Cart management
image

### CODE REVIEW

**Review by haarsh24 — APPROVED** [review]
Looks good , you can merge .

File: src/pages/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (bishnoimukesh) [inline src/pages/Home/Home.css:1]: You can remove this file as it is empty

File: src/pages/Products/Products.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    return (
+        <>
+            <div className="spacer-3rem"></div>
+            <div className="filter-products">
+                <div className="grid-20-80">
+                    <Filter />
+                    <div className="products">
+                        <div className="grid-4-column">
+                            {data.map(product => <Card item={product} key={product.id} />)}
+                        </div>
```
Comment (bishnoimukesh) [inline src/pages/Products/Products.js:20]: add spacing between prices

**Review by bishnoimukesh — APPROVED** [review]
Some suggestions are given, rest the code is clean

**Pull request merged**